app/src/main/java/io/gnosis/safe/ui/transactions/details/CreationTransactionDetailsFragment.kt

```diff
@@ -0,0 +1,109 @@
+package io.gnosis.safe.ui.transactions.details
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.Navigation
+import androidx.navigation.fragment.navArgs
+import io.gnosis.safe.R
+import io.gnosis.safe.ScreenId
+import io.gnosis.safe.databinding.FragmentTransactionDetailsCreationBinding
+import io.gnosis.safe.di.components.ViewComponent
+import io.gnosis.safe.ui.base.fragment.BaseViewBindingFragment
+import io.gnosis.safe.ui.transactions.details.view.TxStatusView
+import pm.gnosis.svalinn.common.utils.copyToClipboard
+import pm.gnosis.svalinn.common.utils.openUrl
+import pm.gnosis.svalinn.common.utils.snackbar
+import pm.gnosis.svalinn.common.utils.visible
+import pm.gnosis.utils.asEthereumAddress
+
+class CreationTransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDetailsCreationBinding>() {
+
+    override fun screenId() = ScreenId.TRANSACTIONS_DETAILS
+
+    private val navArgs by navArgs<CreationTransactionDetailsFragmentArgs>()
+
+    private val statusTextRes by lazy { navArgs.statusTextRes }
```

Comment:
can we just pass status and reuse getColorForStatus that we currently have in TxDetailsFragment (we should then move it to TxUtils)

Reply:
I just saw this after I merged.

Reply:
n.p. You could add it to the next pr if this makes sense.